curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md

```diff
@@ -91,24 +109,91 @@ async (getUserInput) => {
 };
 ```
 
-You can make a `GET` request to `/api/users` to get an array of all users. Each element in the array is an object containing a user's `username` and `_id`.
+You can make a `GET` request to `/api/users` to get a list of all users.
+
+```js
+async(getUserInput) => {
+  const url = getUserInput('url');
+  const res = await fetch(url + 'api/users');
+  if(res.ok){
+    assert(true);
+  } else {
+    throw new Error(`${res.status} ${res.statusText}`);
+  };
+};
+```
+
+The `Get` request to `/api/users` returns an array.
+
+```js
+async(getUserInput) => {
+  const url = getUserInput('url');
+  const res = await fetch(url + 'api/users');
+  if(res.ok){
+    const users = await res.json();
+    assert.isArray(users);
+  } else {
+    throw new Error(`${res.status} ${res.statusText}`);
+  };
+};
+```
+
+Each element in the array is an object containing a user's `username` and `_id`.
```

Comment:
As each user-story/test is separate, it helps to be specific - not to implicitly refer to something in another test

```suggestion
Each element in the `GET /api/users` array is an object literal containing a user's `username` and `_id`.
```